solutions/CF Garland.mdx

```diff
@@ -2,13 +2,141 @@
 id: cf-garland
 source: CF
 title: An impassioned circulation of affection
-author: Neo Wang
+author: Neo Wang, Qi Wang, Jesse Choe
 ---
 
 [Official Editorial](https://codeforces.com/blog/entry/52449)
 
+## Solution (Two Pointers)
+
+**Time Complexity:** $\mathcal{O}(nq)$
+
+For each query we can keep a sliding window where no more than $a$ letters in the window are not $b$.
+
+<LanguageSection>
+
+<CPPSection>
+
+### C++ Implementation:
+
+```cpp
+#include <bits/stdc++.h> // see /general/running-code-locally
```

Comment:
replace this part w/ codesnip?

Reply:
Shouldn't it be `CodeSnip{TemplateShort}` as described here?

https://github.com/cpinitiative/usaco-guide/blob/d03f6397709701cc57809068af7cf0af4009cfed/docs/Content%20Documentation.md#code-blocks-and-code-snippets